device-types/APC/AP7940.yaml

```diff
@@ -0,0 +1,117 @@
+---
+manufacturer: APC
+model: AP7940
+slug: apc-ap7940
+part_number: AP7940
+u_height: 0
+is_full_depth: false
+comments: |
+  Rack PDU, Switched, Zero U, 20A, 200/208V, (21)C13 & (3)C19
+  [APC AP7940 Technical Data](https://www.se.com/us/en/product/download-pdf/AP7940)
```

Comment:
Please change this to`https://www.apc.com/us/en/product/download-pdf/AP7940?filename=APC_NetShelter-Switched-Rack-PDUs_AP7940.pdf`